examples/globe_vector.html

```diff
@@ -75,53 +77,79 @@
                 transparent: true,
+                projection: view.tileLayer.extent.crs(),
                 source: kmlSource,
             });
-            view.addLayer(kmlLayer);
-
-            var gpxSource = new itowns.FileSource({
-                    url: 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/ULTRA2009.gpx',
-                    projection: 'EPSG:4326',
-                }, view.tileLayer.extent.crs());
-
-            var gpxLayer = new itowns.ColorLayer('Gpx', {
-                name: 'Ultra 2009',
-                transparent: true,
-                source: gpxSource,
-            });
-            view.addLayer(gpxLayer);
-
-            var ariegeSource = new itowns.FileSource({
-                    url: 'https://raw.githubusercontent.com/gregoiredavid/france-geojson/master/departements/09-ariege/departement-09-ariege.geojson',
-                    projection: 'EPSG:4326',
-                }, view.tileLayer.extent.crs());
 
-            var ariegeLayer = new itowns.ColorLayer('ariege', {
-                name: 'ariege',
-                transparent: true,
-                style: {
-                    fill: 'orange',
-                    fillOpacity: 0.5,
-                    stroke: 'white',
-                },
-                source: ariegeSource,
-            });
-            view.addLayer(ariegeLayer);
+            promises.push(view.addLayer(kmlLayer));
+
+            // Parse and Convert by iTowns
+            promises.push(itowns.Fetcher.xml('https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/ULTRA2009.gpx')
+                .then(function _(gpx) {
+                    var gpxSource = new itowns.FileSource({
+                        fetchedData: gpx,
+                        projection: 'EPSG:4326',
+                        parser: itowns.GpxParser.parse,
+                    });
+
+                    var gpxLayer = new itowns.ColorLayer('Gpx', {
+                        name: 'Ultra 2009',
+                        transparent: true,
+                        source: gpxSource,
+                    });
+
+                    return view.addLayer(gpxLayer);
+                }));
+
+            // Convert by iTowns
+            promises.push(itowns.Fetcher.json('https://raw.githubusercontent.com/gregoiredavid/france-geojson/master/departements/09-ariege/departement-09-ariege.geojson')
+                .then(function _(geojson) {
+                    return itowns.GeoJsonParser.parse(geojson, {
+                        buildExtent: true,
+                        crsIn: 'EPSG:4326',
+                        crsOut: view.tileLayer.extent.crs(),
+                        mergeFeatures: true,
+                        withNormal: false,
+                        withAltitude: false,
+                    });
+                }).then(function _(parsedData) {
+                    var ariegeSource = new itowns.FileSource({
+                        projection: 'EPSG:4326',
```

Comment:
in this case, the projection is necessary?

Reply:
No, but the previous version of `FileSource` has `projection` mandatory, so I kept it like this. But we can change it if you want.

Reply:
Removed because the `FileSource` now reads `parsedData` if present.